Regra de make para crear o .zip do artigo simplificado

deivisCode opened the pull request:
Temos unha versión simplificada da revista, `artigo_simplificado.tex`,
que usa a mesma clase da revista completa, e tamén as mesmas fontes.
Este `modelo` máis simple compartímolo co resto do grupo de edición.
Dada a cantidade de cambios que soemos facer, penso que é comodo engadir
unha regra na `makefile` para poder xuntar os arquivos necesarios para
dito modelo e así compartilos.

Só hai que escribir:
make modelo

Usa o programa `zip` para comprimir os arquivos, tal vez se pode
facer algún chequeo e algún fallback (e.g. 7zip)

Tamén se cambia o .gitignore para evitar problemas a maiores cos
arquivos comprimidos e git

deivisCode committed:
Regra de `make` para crear o .zip do artigo simplificado

Temos unha versión simplificada da revista, `artigo_simplificado.tex`,
que usa a mesma clase da revista completa, e tamén as mesmas fontes.
Este `modelo` máis simple compartímolo co resto do grupo de edición.
Dada a cantidade de cambios que soemos facer, penso que é comodo engadir
unha regra na `makefile` para poder xuntar os arquivos necesarios para
dito modelo e así compartilos.

Só hai que escribir:
make modelo

Usa o programa `zip` para comprimir os arquivos, tal vez se pode
facer algún chequeo e algún fallback (e.g. 7zip)

Tamén se cambia o .gitignore para evitar problemas a maiores cos
arquivos comprimidos e git

deivisCode committed:
Engadido tamén o arquivo `latexmkrc` ao modelo.zip

Pode ser relevante ter o `latexmkrc` no directorio de traballo. Non podo
xurar que programas como Overleaf non o usen, igual sí. E así aseguramos
que a xente compile exactamente co noso mesmo comando de LuaLaTeX, así
como xerar os directorios auxiliares .aux e .pdf

arebla commented:
@deivisCode 
Produce PDF pero da este erro grave:
```
./artigo_simplificado.tex:32: Module citeproc Error: Style 'american-physics-society.csl' not found
```

Vai facer falla engadir o ficheiro da clase das citacións
```
	zip -r modelo.zip fontes/ artigo_simplificado.tex revista.cls latexmkrc american-physics-society.csl
```

Por outro lado, e se compartimos o .zip como un release en lugar de metelo como opción do makefile? Idealmente `artigo_simplificado.tex` debería ter exemplos de como colocar imaxes (algún ficheiro `.jpeg`, etc.), facer citacións (`bibliografia.bib`, `...cls`), etc.

deivisCode committed:
Engadido o arquivo *.cls para as citas bibliograficas

OLLO: uso o glob * en vez do nome concreto para que non teñamos que
cambialo no futuro. Non creo que haxa problema porque non deberíamos
ter máis .cls 's no directorio

deivisCode commented:
Moi mala idea facer contribuions as 2 da mañá, non probei se funcionaba ou no. Engadín o .csl xa (puxen mal a extension no titulo e no corpo da contribucion, pero na makefile está ben).

Sobre a documentación, sí, deberíamos meter exemplos no artigo_simplificado.tex, pero eso para outra contribucion, simplemente non quería mezclalo co cambio da makefile

O das releases pode tar ben, nunca o mirei. Deixoo apuntado na lista de cousas entón
